### PR TITLE
Front etudiant

### PR DESCRIPTION
**Pull request by Thomas905**

### CODE REVIEW

**Review by Carolinecrepin — CHANGES_REQUESTED** [review]
you have to modify the cards so that they are the color of the variable (black past).
and put some padding between the faq title and the faq and between the card title and the cards

**Review by Carolinecrepin — DISMISSED** [review]
yes ok good!

File: templates/etudiant/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    var panel = this.nextElementSibling;
+    if (panel.style.display === "block") {
+      panel.style.display = "none";
+    } else {
+      panel.style.display = "block";
+    }
+  });
+}
+</script>
```
Comment (RomainClair) [inline templates/etudiant/index.html.twig]: You should have your JS script in files

**Review by RomainClair — CHANGES_REQUESTED** [review]

**Review by elodiedaubie — CHANGES_REQUESTED** [review]
Hi!
Great job :+1: , the page looks nice. However, I noticed few things in the code : 
- the naming of files and methods should be in English. You used the french "etudiant" everywhere, it should be "student".
- the name of "etudiant" method in HomeController should be more specific like studentInformations.
- in the template "etudiant" you should include our component instead of repeating the code (like you did for the footer)
- in scss file, the classes naming might be a problem for futures developments because they are too generics
- you have 2 errors with HTML validator

**Review by elodiedaubie — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**